src/main/java/org/tron/core/db/Manager.java

```diff
@@ -145,6 +150,40 @@ public boolean processTrx(TransactionCapsule trxCap) {
     return true;
   }
 
+  private void voteWitnessCount(Transaction trx) {
+    try {
+      if (trx.getParameterList() == null || trx.getParameterList().isEmpty()) {
```

Comment:
Actually, my bad here. I didn't notice that Apache Commons Collections are included into the project, so you can use `CollectionUtils.isEmpty()` which already has null check

Reply:
@KamilZielinski 
We will be happy to accept your PR which  have not  bugs if you wish.
Thank you for your attention.